content/en/docs/chart_template_guide/accessing_files.md

```diff
@@ -125,40 +125,35 @@ The imported functions are:
 ## Glob patterns
 
 As your chart grows, you may find you have a greater need to organize your files
-more, and so we provide a `Files.Glob(pattern string)` method to assist in
+more, and so we provide the `.Files.Glob(pattern string)` method to assist in
 extracting certain files with all the flexibility of [glob
 patterns](https://godoc.org/github.com/gobwas/glob).
 
 `.Glob` returns a `Files` type, so you may call any of the `Files` methods on
 the returned object.
 
-For example, imagine the directory structure:
+For example, to return all Yaml files from all available directories, you have
+multiple options with Glob:
```

Comment:
These examples aren't equivalent, so I think it might be best to give some text for each one about what is going on.  Something along the lines of:

```
Given a directory tree that contains various `yaml` files you could print the file paths:


... example

Alternatively, you could add their contents:

...other example

The same could be done in this simplified form:

...one more example
```